crates/optimism/node/src/txpool.rs

```diff
@@ -309,6 +321,37 @@ where
             )
         }
 
+        // If the EVM is configured, enable interop tx execution.
+        if let Some(evm_cfg) = &self.evm {
+            // Get the current block
+            let block = if let Ok(Some(block)) = self.inner.client().block_by_number_or_tag(alloy_eips::BlockNumberOrTag::Latest) {
+                block
+            } else {
+                // Failure to get the latest block is a critical error.
+                // Transaction validation must be re-tried.
+                // TODO: fix this error we need a custom type
+                return TransactionValidationOutcome::Invalid(
+                    transaction,
+                    InvalidTransactionError::TxTypeNotSupported.into(),
+                )
```

Comment:
write smthg to logs here with the error, good enough for testing

Reply:
Fixed